pkg/cable/vxlan/vxlan.go

```diff
@@ -257,16 +257,27 @@ func (v *vxlan) getVxlanVtepIPAddress(ipAddr string) (net.IP, error) {
 	return vxlanIP, nil
 }
 
-func (v *vxlan) addIPRule() error {
-	if v.vxlanIface != nil {
-		rule := netlink.NewRule()
-		rule.Table = TableID
-		rule.Priority = TableID
+func (v *vxlan) configureIPRule(operation Operation) error {
```

Comment:
Since the same API is present in route_agent, lets move the API to pkg/netlink/netlink.go file and use it here as well as from route_agent codebase. We can modify this API to take the TableID as argument.

Reply:
Yep, that sounds like a good idea.

Do you know if we expect to support rule's parameters other than table Id as arguments?

Reply:
Currently we don't have such requirement.